frontend/views/containers/payments/MonthOverview.vue

```diff
@@ -44,6 +44,12 @@ export default ({
     ProgressBar
   },
   methods: {
+    getDueDate () {
+      return this.dueDateForPeriod(this.currentPaymentPeriod)
+    },
```

Comment:
This is supposed to be a computed property, not a method. Methods and computed properties are different. When computed properties change the UI gets updated automatically. A method cannot be bound to something as a computed property and won't result in UI updates when its return value changes. Check the Vue.js docs for more info.